### PR TITLE
Remove hosts from relays before deletion

### PR DESCRIPTION
**Pull request by alphadose**

### CODE REVIEW

**Review by abhishek9686 — APPROVED** [review]

File: controllers/hosts.go [hunk 1]
```diff
@@ -150,7 +150,30 @@ func deleteHost(w http.ResponseWriter, r *http.Request) {
 		logic.ReturnErrorResponse(w, r, logic.FormatError(err, "internal"))
 		return
 	}
-
+	if currHost.IsRelay {
+		if _, _, err := logic.DeleteHostRelay(hostid); err != nil {
+			logger.Log(0, r.Header.Get("user"), "failed to dissociate host from relays:", err.Error())
```
Comment (0xdcarns) [inline controllers/hosts.go:155]: need to return error response (http response on all err handles)

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Comment by theguy951357** [comment]
will wait on changes

File: controllers/hosts.go (hunk 1 above)
Comment (alphadose) [reply controllers/hosts.go:155]: done

**Review by 0xdcarns — APPROVED** [review]

**Review by mattkasun — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
built server from branch

joined two nodes

relayed one node

deleted relaying node

relaying status is gone

repeated but deleted relayed node instead

relayed status is gone

pass

**Pull request merged**